cmd/geth/verkle.go

```diff
@@ -74,7 +75,8 @@ func checkChildren(root verkle.VerkleNode, resolver verkle.NodeResolverFn) error
 	switch node := root.(type) {
 	case *verkle.InternalNode:
 		for i, child := range node.Children() {
-			childC := child.Commit().Bytes()
+
+			childC := trieutils.MapToScalarFieldBytes(child.Commitment())
```

Comment:
Once https://github.com/ethereum/go-verkle/pull/428 is merged, this becomes `verkle.MapToScalarFieldBytes`